unittests/Sema/SemaFixture.cpp

```diff
@@ -76,6 +76,36 @@ Type SemaTest::getStdlibType(StringRef name) const {
   return Type();
 }
 
+NominalTypeDecl *SemaTest::getStdlibNomicalTypeDecl(StringRef name) const {
```

Comment:
Nit: If this method was `getNominalTypeDecl(Module&, StringRef)` that could be more general for use in the future if there were to be multimodule unit tests. We could even `getStdlibNominalTypeDecl` that gets the stdlib module and call the generic one ... but it is just an idea not really sure it would be useful 
@xedin does this make sense?

Reply:
I think that could be generalized later once we have found a use-case for it...

Reply:
Make sense, thanks :)